client/header/activity-panel/panels/inbox/index.js

```diff
@@ -30,6 +30,24 @@ class InboxPanel extends Component {
 		this.props.updateCurrentUserData( userDataFields );
 	}
 
+	getScreenName() {
```

Comment:
Could this live in `/inbox/card.js` rather than having to pass the screen name in as a prop? It feels like the logic is spread out a little.

Reply:
I have my doubts about moving this code inside the card since this calculation would be processed every time we create a card. (E.g. if we have 100 cards this calculation would be done 100 times).
I thought that it would be more efficient to keep this logic outside.
Do you think the way it is now is a considerable problem?

Reply:
It feels strange to have the logic here. I don't think running the function 100 times would be a performance hit, it's not doing a lot.

Reply:
I agree with @becdetat - it seems odd to pass this down as a prop solely for event tracking purposes. Is this a valuable enough value to include in all Tracks events? If so - let's add it to our `recordEvent` wrapper instead. _cc: @pmcpinto_

Reply:
Thank you @becdetat and @jeffstieler. The suggestion makes sense.
I made this change in the commit [a2815bc](https://github.com/woocommerce/woocommerce-admin/pull/4320/commits/a2815bc64b288adcb098a6fa473494c197378010)

Reply:
> Is this a valuable enough value to include in all Tracks events?

Sorry for the late reply on this. At this point it will be mainly useful in events related to the inbox panel, so it doesn't need to be included in all Tracks events. Thanks